requirements.txt

```diff
@@ -1,18 +1,8 @@
-backports.lzma==0.0.8
-click==6.7
-cryptography==2.3
+click>=6.7
 cloud-volume>=0.38.0
 google-cloud-logging
-google-api-python-client==1.6.2
-networkx==2.1
 numpy>=1.14.2
-pyasn1
-pyopenssl>=17.5.0
 pytest>=3.3.1
-scipy>=1.1.0
 task-queue>=0.4.1
-tenacity>=4.10.0
 -e git+https://github.com/seung-lab/tqdm.git#egg=tqdm
-oauth2client==3.0.0
-protobuf>=3.5.1
-python-jsonschema-objects==0.3.3
```

Comment:
will this work after removing the specific version?

Reply:
The `python-jsonschema-objects==0.3.3` was just recently added here to avoid the cloud-volume issue we ran into over the weekend. See https://github.com/seung-lab/cloud-volume/issues/179
It shouldn't be here in the first place, as it is a cloud-volume dependency, not a _direct_ igneous dependency.